modules/BackToStart/src/backtostart.cpp

```diff
@@ -26,6 +27,17 @@ void BackToStart::onReturnTrajectoryRequest(const std::shared_ptr<atos_interface
         response->success = false;
         return;
     }
+
+	// Get the turn radius of the williamson turn.
+	double turnRadius;
+	auto success = this->get_parameter("turnRadius", turnRadius);
+	if (!success) {
+		throw std::runtime_error("Could not read parameter turnRadius. Using 5m as default.");
+	}
+	else {
+        turnRadius = 5.0;
+	}
```

Comment:
This logic looks like it is inverted.
If we fail to get the turnRadius param, we throw an exception with the text "using 5m as default" and exit the current function.
If we successfully get the turnRadius parameter, we ignore it, and forcefully set turnRadius to 5.0.

Reply:
The default value for turn radius is set when declaring the parameter, meaning this isn't necessary. Or am I misunderstanding what's happening here?